pkg/k8shandler/fluentd.go

```diff
@@ -423,7 +461,7 @@ func (clusterRequest *ClusterLoggingRequest) updateFluentdDaemonsetIfRequired(de
 				return fmt.Errorf("Timed out waiting for Fluentd to be ready")
 			}
 		}
-		logger.DebugObject("updating fluentd to: %v", current)
+		logger.DebugObject("updating fluentd to: %v", desired)
```

Comment:
i think this should be `current` based on the code above we are updating the `current` definition... but below it updates `desired`.

The idea behind updating `current` is that we have all the updated k8s metadata in the `current` definition and we would update its spec to match `desired`... not sure where this drift happened